DEVELOPMENT.md

```diff
@@ -255,6 +259,8 @@ to start a debugger to which attach from your IDE.
     * [Goland instructions](https://golangforall.com/en/post/go-docker-delve-remote-debug.html#goland-ide)
 1. Generate traffic:
     * API: Query the API using `curl` or your browser:
+
+   **NOTE**: For this to work the `test/users/test-user.yaml` must be applied
```

Comment:
You need a period at the end of the sentence:

```suggestion
   **NOTE**: For this to work the `test/users/test-user.yaml` must be applied.
```

Reply:
Fixed.